pkgs/by-name/li/libguytools/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchurl
+, libsForQt5
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libguytools";
+  version = "2.1.0";
+
+  src = fetchurl {
+      url = "mirror://sourceforge/project/libguytools/libguytools/LatestSource/tools-${finalAttrs.version}.tar.gz";
+      hash = "sha256-eVYvjo2wKW2g9/9hL9nbQa1FRWDMMqMHok0V/adPHVY=";
+  };
+
+  qmakeFlags = [
+    "trunk.pro"
+    "toolsstatic.pro"
+  ];
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ libsForQt5.qmake libsForQt5.wrapQtAppsHook ];
+  dontWrapQtApps = true;
```

Comment:
why?

Reply:
@eclairevoyant If I remove that, I get the following error:
```
Running phase: qtPreHook
Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
error: builder for '/nix/store/fncpc8jyf7b1h08yipf25pkmls71pxhn-libguytools-2.1.0.drv' failed with exit code 1;
       last 10 log lines:
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/qqzz17mjs51kpfx9faw98f35r7ffwqng-xcb-util-renderutil-0.3.10-dev
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/7rd4b08axmpg5nnys8kkjbmj7cxlkdlc-xcb-util-renderutil-0.3.10
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/x3l69111mbfd38dd8q59hqv94vz6d6jn-xcb-util-wm-0.4.2-dev
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/fiwwkqgfgb4g2f63iycpsx2h5amk2jdv-xcb-util-wm-0.4.2
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/h5j6ggf0g230bavqarsvyi23zw789fv3-libglvnd-1.7.0-dev
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/d6gvgzzifggrb7fh1v0yi8bvrdlwhpqa-libglvnd-1.7.0
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/hjspqdvym6v553582mxp2a3f84p4aqyr-qtbase-5.15.14-bin
       > calling 'envTargetTargetHook' function hook 'make_glib_find_gsettings_schemas' /nix/store/9wba9diw8b199hfzcv8w99wzma66dzm1-qtbase-5.15.14
       > Running phase: qtPreHook
       > Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
       For full logs, run 'nix log /nix/store/fncpc8jyf7b1h08yipf25pkmls71pxhn-libguytools-2.1.0.drv'.
```
I can also use only:
```
nativeBuildInputs = [ libsForQt5.qmake libsForQt5.wrapQtAppsHook ];
```
and remove `dontWrapQtApps`. What should I use?